neo4j/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # CHANGELOG - neo4j
 
+## 3.0.3 / 2024-03-29
+
+***Added***:
+
+* Added additional bolt, page_cache eviction and relationship type  metrics which have been introduced since Neo4j 5.10
+
+***Changed***:
```

Comment:
Can you move the changed section to above the added section? That's currently failing validation right now.